scripts/ci/do_not_merge.py

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env python3
+
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import os
+import sys
+
+import github
+
+DNM_LABELS = ["DNM", "DNM (manifest)", "TSC", "Architecture Review", "dev-review"]
+
+
+def parse_args(argv):
+    parser = argparse.ArgumentParser(
+        description=__doc__,
+        formatter_class=argparse.RawDescriptionHelpFormatter,
+        allow_abbrev=False,
+    )
+
+    parser.add_argument("-p", "--pr", required=True, type=int, help="The PR number")
```

Comment:
Is having `-p` and `--pr` really that useful? Maybe use `-pr` and `--pull-request`? (Same as `west patch gh-fetch`)

Reply:
what was I thinking, this makes no sense... changed to `-p` and `--pull-request`